denizen_scripts/survival/repo-link/spawn_scripts/spawn_npc_grim.dsc

```diff
@@ -79,7 +77,7 @@ grim_steal_prevention:
             - narrate "<&c>You do not have a death location for me to show you."
             - inventory close
           - else:
-            - narrate "<&a>You death location is <player.flag[player_death_location].as_location.simple.formatted||not set>"
+            - narrate "<&a>Your death location is <player.flag[player_death_location].as_location.simple.formatted||not<&sp>set>"
```

Comment:
Would recommend using a plain space instead of `<&sp>` only because the argument here is wrapped in quotes

Reply:
```suggestion
            - narrate "<&a>Your death location is <player.flag[player_death_location].as_location.simple.formatted||not set>"
```